src/common/i18n/index.ts

```diff
@@ -0,0 +1,10 @@
+import { MessageFormatElement } from 'intl-messageformat-parser';
+
+import messages_en from "./en.json";
```

Comment:
run lint on this file (" and ' in the same file)

Reply:
actually lint was run, nothing about it, strange.
Fixed